gridspeccer/core.py

```diff
@@ -89,10 +89,10 @@ def make_figure(name):
     except AttributeError:
         pass
 
-    for k, axes in axes.items():
+    for k, axis in axes.items():
```

Comment:
Actually, `ax` would be a more fitting name, because we do iterate over [Axes](https://matplotlib.org/api/axes_api.html#the-axes-class)-objects that themselves contain two or more [Axis](https://matplotlib.org/3.3.3/api/axis_api.html?highlight=axis#matplotlib.axis.Axis) objects.

I fully agree that we probably should change the double-occurrence of `axes`.

Reply:
I completely agree, but `ax` is not pep8 compatible: variable names (other than ijk i think) have to be longer than 2 letters. If you search the older log for 'ax' you'll see the messages. one can alter the regex used for this (like here i think http://pylint.pycqa.org/en/1.9/user_guide/options.html#cmdoption-variable-rgx)

not sure what to do. The first time I encountered this using `axis` was straight forward, but there are occurences where `ax` would be beneficial

Reply:
now that I've looked for it again I found [this](https://stackoverflow.com/a/35182977), so one could say that `ax` is a good name. but would have to revert all `axis` changes...
your opinion?

Reply:
Permitting `ax` as a good name seems like a good solution :+1: